src/extension.ts

```diff
@@ -295,6 +295,9 @@ class KeyStatusManager {
         }
 
         this.#logger.info(`Try to unlock current key: ${theKey.fingerprint}`);
+        // Escape percent character.
+        // https://www.gnupg.org/documentation/manuals/assuan/Client-requests.html#Client-requests
+        passphrase.replace(/%/g,'%25');
```

Comment:
I'm so sorry, I just realized I messed this up. The code I submitted doesn't exactly match
This needs to be
```javascript
passphrase = passphrase.replace(/%/g,'%25');
```